kolibri/plugins/learn/assets/src/vue/content-render.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
It's also fine to leave it in Learn until the coach stuff comes along - I think being conservative with adding things to core is good practice